packages/text-input/src/TextInput.tsx

```diff
@@ -107,6 +117,8 @@ interface TextInputProps extends HTMLElementProps<'input', HTMLInputElement> {
   handleValidation?: (value: string) => void;
 
   ['aria-labelledby']?: string;
+
+  sizeVariant?: SizeVariantType;
```

Comment:
I think we just call this prop `size` to keep in line with other components

Reply:
I tried to use `size` but it looks like since we're using the `HTMLInputElement` interface that already has a property of `size` with a type of `number` so I was getting a TS error. To avoid that I changed it to `sizeVariant`.